application/lib/ui/new_event_screen_ui.dart

```diff
@@ -9,8 +19,174 @@ class NewEventScreen extends StatefulWidget {
 }
 
 class _NewEventScreenState extends State<NewEventScreen> {
+  final _formKey = GlobalKey<FormState>();
+  final dateController = TextEditingController();
+  File? _imageFile;
+  String? selectedCategory;
+
+
   @override
   Widget build(BuildContext context) {
-    return Text(newEvent);
+    final newEventScreenViewModel =
```

Comment:
Itt pedig a new event sem ehhez a mergerequesthez tartozik. Kérlek ezt is vonjuk vissza.